pulser/sequence.py

```diff
@@ -1111,6 +1128,45 @@ def _reset_parametrized(self) -> None:
         self._variables = {}
         self._to_build_calls = []
 
+    def config_slm_mask(self, qubits: Set[QubitId]) -> None:
```

Comment:
Don't forget you need to store calls to this function for serialization purposes

Reply:
You mean decorating with `@_store`?
I'm still not sure when to do that. Is it because some of its values depend on pulses?

Reply:
Yeah, it will depend on whether we want to allow parametrized objects to serve as qubit ID's. I would say no for now, so we should talk about how to do it.